ontology/nva-type.ttl

```diff
@@ -0,0 +1,107 @@
+@prefix nva: <https://api.nva.unit.no/ontology/nva#> .
+@prefix nva-type: <https://api.nva.unit.no/ontology/nva-type#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+nva-type: a owl:Ontology ;
+  dcterms:title "The NVA type ontology" ;
+  dcterms:description "The type definitions used in the APIs used in NVA" ;
+  dcterms:creator "Unit" ;
+  dcterms:publisher "Unit" ;
+  dcterms:license <https://creativecommons.org/publicdomain/zero/1.0> .
+
+nva-type:ClothingDesign a nva:InstanceType ;
+  rdfs:isDefinedBy nva-type: ;
+  rdfs:label "Clothing design"@en ;
+  rdfs:label "Klesdesign"@nb ;
+  rdfs:label "Klesdesign"@nn ;
+  rdfs:comment "A design for clothing"@en ;
+  rdfs:comment "En design for klær"@nb ;
+  rdfs:comment "Ein design for klær"@nn .
```

Comment:
For meg høres "et design" langt mer naturlig ut enn "en design", men virker som hankjønn er minst like rett 🤯 https://www.sprakradet.no/svardatabase/sporsmal-og-svar/designen-eller-designet/